app/assets/javascripts/countdown.js

```diff
@@ -0,0 +1,38 @@
+function Countdown(trackingElements) {
+  this.trackingElements = trackingElements;
+  this.auction_end_time = trackingElements.auction_end_time.text();
+
+  this.ongoing = function(durationCalculator) {
+    this.trackingElements.days.html(durationCalculator.days);
+    this.trackingElements.hours.html(durationCalculator.hours);
+    this.trackingElements.minutes.html(durationCalculator.minutes);
+    this.trackingElements.seconds.html(durationCalculator.seconds);
+  };
+
+  this.onEnd = function(durationCalculator) {
+    this.trackingElements.auction_end_time.html("Auction for this item is closed");
```

Comment:
By putting this here, your countdown object is still auction specific. I wan to able to register anything to be done when the countdown end. As you'll see in my original example, this means that your onEnd method takes a function to be called when the countdown ends.